scss-test.scss

```diff
@@ -0,0 +1,8 @@
+.parent {
+  color: #FFFFFF;
+  background: red;
+
+  .child {
+    color: #aaa;
```

Comment:
Color literals like `#aaa` should only be used in variable declarations; they should be referred to via variable everywhere else.